app/controllers/patients_controller.rb

```diff
@@ -368,6 +368,9 @@ def update_fields(patient, params, household, propagation)
       patient.closed_at = DateTime.now
     end
 
+    # Do not allow continuous exposure to updated for closed records
+    params_to_update.delete(:continuous_exposure) if params_to_update.include?(:continuous_exposure) && !patient.monitoring
```

Comment:
Interested in hearing why this (deleting `continuous_exposure` if in `params_to_update`) instead of setting it to false (and force overwriting regardless of current state)

Reply:
Seemed clearer. We don't want to update that field at all in this case, so removing from `params_to_update` made the most sense. The alternative seemed like more steps than this, which reads nicely as well. But yeah either would work.

Reply:
Makes sense!